socorro/unittest/config/commonconfig.py.dist

```diff
@@ -25,7 +25,7 @@ oldDatabaseName.default = 'socorro_test'
 
 databaseUserName = cm.Option()
 databaseUserName.doc = 'the user name for the database servers'
-databaseUserName.default = 'breakpad_rw'
+databaseUserName.default = 'test'
```

Comment:
This is legacy stuff, but until it is fully deprecated, I think we should just use 'test' as the user, since this is what we are using for the rest of our unit test configs.